examples/fashion_conv_dense_test.py

```diff
@@ -101,14 +101,14 @@
 
 model = ConvNet(
     name='conv_net_test',
-    input_network=None,
+    input_networks=None,
     dimensions=(1, 28, 28),
     config=conv_net_config,
 )
 
 model1 = DenseNet(
     name='dense_net_test',
-    input_network=model,
+    input_networks=[model],
```

Comment:
I thought we were using another dict method of passing models with options or would you rather leave it to the user to explicitly specify input network functionality after building the multi-input? e.g. model.freeze_input_networks()

Reply:
there's also no need to do this; we could allow the user to pass multiple types and just use .isinstance() and then convert as necessary.